src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Usage/DetectPLINQNops.Fixer.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.NetCore.Analyzers.Usage
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = DetectPLINQNopsAnalyzer.RuleId), Shared]
+    public sealed class DetectPLINQNopsFixer : CodeFixProvider
```

Comment:
Either this is meant to be covering both C# and VB.NET in which case it's missing from the `ExportCodeFixProvider` attribute or this is indeed C# specific in which case it needs to be renamed and moved to the CSharp project.

Reply:
I still have to convert the fixer to the ioperation style, the fixer is not as desired atm